templates/taxi/driver_list.html

```diff
@@ -8,6 +8,11 @@ <h1>
       </a>
     </h1>
 
+    <form method="GET" action="">
+        <input type="text" name="q" placeholder="Search by username">
+        <button type="submit">Search</button>
+    </form>
```

Comment:
Looks like it is duplication in all files. Create separate template and use it where you need